### DISCUSSION TITLE
[Question]: Audio detection & audio event storage only (and just send event to HA)?

### DISCUSSION THREAD

**Discussion by joergbattermann** [blakeblackshear/frigate#10763] [Ask A Question]
### What is your question:

Does anyone know if I can use & configure Frigate to _only_ run certain audio detectors for one or more cameras, store the events as audio-only (no video wanted/needed.. and probably just for debugging purposes) and send those detected audio events (i.e. 'fire_alarm' or 'bark') to HA?

Basically, I want to start with Frigade by being 'just' a smart audio detector to trigger certain automations on the HA side of things thereafter and (in the beginning) keep/store these events on Frigade side of things only the mere audio, not the video stream.

The 2nd step, once I know if/how well it works, would be to discard and NOT store any event 'footage' in/with Frigade at all & just notify HA when these events happen.

Is that possible or are there currently limitations that would prevent one or the other? I'd probably want to go with a Raspberry Pi5 + Coral TPU for the detection side of things & cameras would be Unifi ones.

Thanks!

**Comment by NickM-27** [comment]
I am a bit confused on the request, if your end goal is to only use audio for notifications in HA and not save events for it then that is easy to do. Only saving audio clips and not video is not supported

**Reply by joergbattermann** [reply]
Thanks @NickM-27. I was 'imaging' (hence the question) that one could potentially configure the underlying ffmpeg call [to save the event as as .m4a / audio only](https://stackoverflow.com/a/66120164/2591).

Would you happen to have any direct pointers for the easy, first part ("_only use audio for notifications in HA and not save events for it_")?

Thanks!

**Reply by NickM-27** [reply]
if you don't want to do other object detection on that camera then simply not enabling recording or snapshots will do it. 

Otherwise, you'll need to set 

```
record:
  events:
    retain:
      objects:
        - fire_alarm: 0
```

for example
